ctapipe/tools/quickstart.py

```diff
@@ -55,13 +56,14 @@
 ctapipe-process --help-all
 ```
 
-## ctapipe-train-energy-regressor / ctapipe-train-particle-classifier configs
+## ctapipe-train-energy-regressor / ctapipe-train-particle-classifier / ctapipe-train-disp-reconstructor configs
 
 Included here are also base configurations for training machine learning (ML)
-models for energy regression and gamma/hadron separation.
+models for energy regression, gamma/hadron separation and disp origin reconstruction.
```

Comment:
I think we should add proper configuration files for use with ctapipe-quickstart, not the unit test files with slimmed down simplified configurations as we have them here.

Ok for now, but maybe add a note that these files are optimized for very fast training for demos and unit tests and will not result in good performance?